app/src/main/java/ch/epfl/sweng/SDP/game/VotingPageActivity.java

```diff
@@ -226,34 +225,10 @@ public void startHomeActivity(View view) {
             }
         }
 
-        if (rankingFragment != null) {
-            createAndStoreGameResult();
-        }
         launchActivity(HomeActivity.class);
         finish();
     }
 
-    private void createAndStoreGameResult() {
-        List<String> rankedUsernames = rankingFragment.getRankedUsernames();
-        int rank = rankedUsernames.indexOf(username);
-        Bitmap drawing = null;
-
-        for (int i = 0; i < playersNames.length; i++) {
-            if (playersNames[i] != null && playersNames[i].equals(username)) {
-                drawing = drawings[i];
-            }
-        }
-
-        // TODO: replace constants and drawing by the good values
-        if (drawing != null) {
-            GameResult gameResult = new GameResult(rankedUsernames, rank, 23, -5,
-                    drawing, this);
-            LocalDbHandlerForGameResults localDb =
-                    new LocalDbHandlerForGameResults(this, null, 1);
-            localDb.addGameResultToDb(gameResult);
-        }
-    }
-
```

Comment:
Why would you delete that ?

Reply:
I think @Claudio5  is implementing this on the server side

Reply:
Due to field envy this method has been moved to the RankingFragment class